recipes-devtools/clang/clang-cross-canadian_git.bb

```diff
@@ -12,7 +12,7 @@ PN = "clang-cross-canadian-${TRANSLATED_TARGET_ARCH}"
 require clang.inc
 inherit cross-canadian
 
-DEPENDS += "nativesdk-clang binutils-cross-canadian-${TRANSLATED_TARGET_ARCH} virtual/${HOST_PREFIX}binutils-crosssdk virtual/nativesdk-${HOST_PREFIX}libc-for-gcc"
+DEPENDS += "nativesdk-clang binutils-cross-canadian-${TRANSLATED_TARGET_ARCH} virtual/${HOST_PREFIX}binutils-crosssdk"
```

Comment:
We can not drop it but replace it with `virtual/nativesdk-libc`

Reply:
Updated